src/content/development/contribution-guide/_index.en.md

```diff
@@ -10,23 +10,30 @@ in order for their contributions to the project to be addressed in a timely mann
 
 ## Project Resources
 
-To be expanded
+Submariner uses [GitHub projects](https://docs.github.com/en/issues/organizing-your-work-with-project-boards) to manage a particular release.
+Read [here](https://docs.github.com/en/issues/organizing-your-work-with-project-boards/tracking-work-with-project-boards) to know more on
```

Comment:
```suggestion
Read “[Tracking progress on your project board](https://docs.github.com/en/issues/organizing-your-work-with-project-boards/tracking-work-with-project-boards)” to learn
```

It’s better to make link titles explicit.

Reply:
(for accessibility reasons at least)